darwin/importer/importer.py

```diff
@@ -318,12 +352,104 @@ def _update_payload_with_properties(
             annotation["annotation_properties"] = dict(_map)
 
 
+def _update_payload_with_item_level_properties(
+    item_property_values, client, dataset, import_annotators, import_reviewers
+) -> List:
+    """
+    Adds item-level properties to the annotation import payload if any are present
+
+    Args:
+        item_property_values
+        cliennt
+        dataset
+    """
+    if not item_property_values:
+        return []
+
+    serialized_item_level_properties: List[Any] = []
+    actors: List[dt.DictFreeForm] = []
+    # Get team properties
+    _, team_item_properties_lookup = _get_team_properties_annotation_lookup(
+        client, dataset.team
+    )
+    for item_property_value in item_property_values:
+        item_property = team_item_properties_lookup[item_property_value["name"]]
+        item_property_id = item_property.id
+        item_property_value_id = next(
```

Comment:
not sure how many we expect, but if is many might be worth making a lookup for this